.github/workflows/go.yml

```diff
@@ -12,6 +12,8 @@ on:
 
 env:
   EACHMODULE_CONCURRENCY: 2
+  SMITHY_GO_REPOSITORY: ${{ vars.SMITHY_GO_REPOSITORY }}
```

Comment:
how is `vars.SMITHY_GO_REPOSITORY` being set?

Reply:
GitHub has support for env vars at the repo level (org level as well, I believe)